src/OpenTelemetry.Exporter.Jaeger/Implementation/IJaegerClient.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace OpenTelemetry.Exporter.Jaeger.Implementation
 {
-    public interface IJaegerUdpClient : IDisposable
+    public interface IJaegerClient : IDisposable
```

Comment:
Do we need to rename the interface? It would be a break change to anyone who's used the public interface.

Reply:
No reason, really. Overstepping my bounds ;) It just seemed to add more value with this name. It's not representing anything UDP specific, just any old client that can be plugged into `JaegerThriftClientTransport` (so long as it can satisfy the signature). Seemed like if you were going to switch out the client it would because you wanted to change the protocol, not re-implement UDP. But happy to change it back, just confirm for me please. Still alpha so not sure if there is an opportunity to be breaking.

Reply:
@MikeGoldsmith I don't think it's critical to keep names if it's reasonable rename. Have you seen anybody taking dependency on it already?

Reply:
Yeah, I don't think there is anyone who depends on it. I think I queried it just because it was mixed in with the other changes.

I'm not against it and think the updated name is better / clearer.